_plauscherl/78.html

```diff
@@ -26,8 +26,23 @@
 sponsor: <a href="https://cloudflight.io">Cloudflight</a>
 speakers:
 -
-   name: Someone
-   talk: "Something Interesting"
+   name: Stefan Starke
+   talk: "Azure yourself, there is a lot to learn"
    img: "/img/speaker_no_image.jpg"
    language: en
+-
+   name: Christian Olear
+   talk: "Crawler crwl.io"
+   img: "/img/speaker_no_image.jpg"
+   language: en
+-
+   name: Martin Forstner
+   talk: "Accessibility - Essential for some. Useful for all"
+   img: "/img/speaker_no_image.jpg"
+   language: en
+-
+   name: David Tanzer
+   talk: "Contract testing mit Pact"
+   img: "/img/speaker-avatars/david_tanzer.jpg"
+   language: de
```

Comment:
David will do the talk in English as well

Reply:
was misled by the German titel :)